api/src/opentrons/protocol_engine/types.py

```diff
@@ -398,3 +407,17 @@ class HeaterShakerMovementRestrictors:
     plate_shaking: bool
     latch_closed: bool
     deck_slot: int
+
+
+class CalibrationPosition(str, Enum):
+    """Deck slot to move to."""
+
+    PROBE_POSITION = "probePosition"
+    ATTACH_OR_DETACH = "attachOrDetach"
+
+
+class CalibrationCoordinates(NamedTuple):
+    """Calibration coordinates and critical point."""
+
+    coordinates: Point
```

Comment:
Use the engine-specific `DeckPoint` to represent absolute coordinates in the engine rather than `opentrons.types.Point`

```suggestion
    coordinates: DeckPoint
```

Reply:
The reason I choose Point is bc `get_slot_center_position` is returning `Point`. and hardware_api.move_to is expecting `Point`